tests/jballerina-unit-test/src/test/resources/test-src/types/xml/xml-literals-with-namespaces.bal

```diff
@@ -166,3 +166,32 @@ function testXmlLiteralUsingXmlNamespacePrefix() {
         panic error("Assertion error", expected = expectedStr, found=s);
     }
 }
+
```

Comment:
I think this test cases should be in xmllib_test.bal as this is about `xml:createElement` rather than xml literal.